modules/dcache-vehicles/src/test/java/diskCacheV111/vehicles/JobInfoTests.java

```diff
@@ -0,0 +1,74 @@
+package diskCacheV111.vehicles;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.*;
+
+/**
+ * Created by alenaschemmert on 03.08.17.
+ */
+
+public class JobInfoTests {
+
+    private JobInfo jobInfo;
+    private OldJobInfo oldJobInfo;
+    private String ref;
+
+    @Before
+    public void setUp() {
+        jobInfo = new JobInfo(System.currentTimeMillis(), System.currentTimeMillis() + 1000, "working", 1, "waldo", 2407);
```

Comment:
As if to underline this point: the unit test just failed when I trigged our CI system:

https://ci.dcache.org/job/dCache-master/7613/testReport/junit/diskCacheV111.vehicles/JobInfoTests/OldAndNewShouldBeEqual/

Seemingly, the seconds "rolled over" between creating JobInfo and OldJobInfo, so the output was different (failing the test) although the code is basically sound.